database/database.go

```diff
@@ -0,0 +1,23 @@
+package database
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func Connect(connection string) (*sql.DB, error) {
+	db, err := sql.Open("postgres", connection)
+
+	if err != nil {
+		panic(err)
+	}
+
+	err = db.Ping()
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Println("Successfully connected!")
```

Comment:
Use log package for logging.

log.Println(...)

Reply:
Update logging.